src/CompletionProvider.php

```diff
@@ -382,18 +376,17 @@ public function provideCompletion(PhpDocument $doc, Position $pos): CompletionLi
      * @param string[] $fqns
      * @return string[]
      */
-    private function expandParentFqns(array $fqns): array
+    private function expandParentFqns(array $fqns)
```

Comment:
Because it takes an array, not a single definition. There would be identical code looping through the array in both places where this method is used if it is removed. Let me know if you want me to remove it.